requirements.txt

```diff
@@ -1,5 +1,5 @@
 cryptography==37.0.4
```

Comment:
I think updating cryptography breaks pip and the installation of Patroni, so I've only updated ops for the time being.